run-build.sh

```diff
@@ -93,6 +96,14 @@ while [[ $# > 0 ]]; do
             args=( "${args[@]/$2}" )
             shift
             ;;
+        --linux-portable)
+            # Pass in a specific version (the one we set up to bootstrap this experience). Once 
+            # we have an official build we can remove the version parameter and use the latest.
+            LINUX_PORTABLE_INSTALL_ARGS="--linux-portable --version 2.0.0-alpha-005151"
+            # Until we get test support for 2.0 we need to pass in the targets without test.
+            LINUX_PORTABLE_BUILD_ARGS="/p:Rid=\"linux-x64\" /p:OSName=\"linux\" /p:IncludeAdditionalSharedFrameworks=false /p:CLITargets=\"Prepare;Compile;Package;Publish\""
```

Comment:
Can we rename `LINUX_PORTABLE_BUILD_ARGS` to be something more general?  That way if other things need to specify build args, they can use the same variable?